### DISCUSSION TITLE
NUC8i7HVK can't be insallted any more

### DISCUSSION THREAD

**Discussion by clicx** [osy86/HaC-Mini#961] [Q&A] [answered]
**Describe the bug**
many days ago I have installed Catalina with successfully. But after a boot failure I can't install again.
When I boot from USB disk, it shows the partition to boot, after selected the USB drive, the boot menu shows correctly with Install Catalina and Reset NVRam. After select Install Catalina, windows go dark screen and reboot after a few seconds. I have tried many ways always the same problem.
And after the second boot, the bootmenu in the bios of USB drive shows as "OpenCore", so I think it related to the Opencore.

**Expected behavior**

**Environment:**
 - Running OSX version: macOS Cataina 10.15.7
 - Target drive OSX version: 
 - HaC Mini installer version: newest

**Additional context**

**Logs**
Please attach `/var/log/install.log`

**Comment by smounives** [comment]
I have the same issue. I have tried installing Big Sur and Monterey, both are normal, only Catalina will repeatedly enter the USB drive Installer after reboot.

**Comment by clicx** [comment]
Hi smounives, is Big Sur working good on the hades?

**Comment by smounives** [comment]
> Hi smounives, is Big Sur working good on the hades?

yes

**Comment by x2358473** [comment]
I hope someone can fix it，Catalina has a better experience than Big Sur.

**Comment by Murphy41** [comment]
I have the same issue. The boot menu in the bios of the USB drive shows as "OpenCore" and it will get back to the boot menu every time after I chose "OpenCore". I've tried tapping or holding alt but still doesn't work.

**Comment by osy** [comment]
Try to reset NVRAM. More troubleshooting tips: https://osy.gitbook.io/hac-mini-guide/post-installation/support#troubleshooting

**Comment by clicx** [comment]
I have tried reset NVMRAM in the boot menu for many time, and also reset on the motherboard, but results are same, the OpenCore will appear again. However Big Sur can boot.

**Reply by osy** [reply]
Are you on SATA or NVMe?

**Reply by clicx** [reply]
NUC8i7HVK have NVMe attached, but this is boot period, just can't boot to installer.

**Reply by osy** [reply]
Do you have "SATA support (macOS 11+)" checked or unchecked in the installer? If you have it checked, can you try unchecked?

**Comment by osy** [comment]
Are you guys enabling the boot security feature of HaC mini?

**Reply by clicx** [reply]
No, that was disabled.

**Reply by osy** [reply]
First try above (uncheck SATA support). If that still doesn't work, can you try installing every version of https://github.com/osy/HaC-Mini/releases in reverse until you get to one that works? You can re-install over your USB installation drive without having to recopy macOS installer files.

**Reply by clicx** [reply]
Thanks, I will try it.

**Comment by osy** [comment, marked as answer]
Should be fixed in v3.1.4
